File: C++/1075.cpp
// Problema: 1075 - Resto 2 | Resposta: Accepted
// Linguagem: C20++ [+0s]   | Tempo: 0.000s

#include <iostream>
#define MAX 10000

int main() {
    int divisor;
    std::cin >> divisor;

    for (int i = 2; i < MAX; i += divisor) {
        std::cout << i << '\n';
    }

    return 0;
}

File: PHP/1075.php
<?php
// Problema: 1075 - Resto 2    | Resposta: Accepted
//Linguagem: PHP (8.1.2) [+1s] | Tempo: 0.163s

$divisor = intval(trim(fgets(STDIN)));
$MAX = 10000;
for ($i = 0; $i < $MAX; $i++) {
    if ($i % $divisor === 2) {
        echo $i . PHP_EOL;
    }
}
?>